integration/ct_integration_test.sh

```diff
@@ -1,62 +1,113 @@
 #!/bin/bash
 set -e
-INTEGRATION_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-. ${INTEGRATION_DIR}/common.sh
+INTEGRATION_DIR="$( cd "$( dirname "$0" )" && pwd )"
+. "${INTEGRATION_DIR}"/common.sh
 
-RPC_PORT=36962
-CT_PORT=6962
+# Default to one RPC server and one HTTP server.
+RPC_SERVER_COUNT=${1:-1}
+HTTP_SERVER_COUNT=${2:-1}
+BASE_RPC_PORT=36961
+BASE_HTTP_PORT=6961
+LB_PORT=46962
 
-. ${INTEGRATION_DIR}/ct_config.sh
+for i in $(seq 1 ${RPC_SERVER_COUNT})
```

Comment:
Suggestion: Simplify to:

for ((i=0; i < RPC_SERVER_COUNT; i++))); do ...; done